cloudformation-templates/pattern-1/testgrid.yaml

```diff
@@ -0,0 +1,36 @@
+# TestGrid configuration file.
+version: "0.9"
+infrastructureConfig: 
+  containerOrchestrationEngine: None
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  parameters: 
+    - 
+      JDK: ORACLE_JDK8
```

Comment:
@azinneera Can we remove the default values?

Reply:
Removed in e733d2d